### PR TITLE
bug: baseline to cryptography 1.8.2

### PR DESCRIPTION
**Pull request by jrconlin**

### CODE REVIEW

**Comment by safwanrahman** [comment]
1.9 break ``vapid`` right? @jrconlin

**Comment by jrconlin** [comment]
Already updated vapid to use 1.9 (and found a fun bug with public key validation)
https://github.com/web-push-libs/vapid/pull/52

Also created https://github.com/martinthomson/encrypted-content-encoding/pull/39

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-cryptography==1.8.2
+cryptography>=1.8.1,<1.10
```
Comment (safwanrahman) [inline requirements.txt]: in ``vapid`` we are using ``cryptography>=1.8.2,<=1.10``. So this should not be same?

Comment (jrconlin) [inline requirements.txt]: Well, the prior version of http-ece is limited to 1.8.1. vapid doesn't use http-ece, so we're at liberty over there. 

martin landed the patch to move to 1.9, but hasn't promoted the package on pypi yet.

**Pull request merged**